packages/waku/package.json

```diff
@@ -34,6 +34,10 @@
       "types": "./dist/internals.d.ts",
       "default": "./dist/internals.js"
     },
+    "./hono": {
```

Comment:
Just curious here, but why do `/hono` and `/internals` need to be in the published package json exports if they are for internal use only?

Reply:
I did it at first (locally without commits), but it feels like mixed. `/internals` are "more internal" than `/hono`, so to speak. 😄 
In any case, I might be reconsidering it in the future, before finalizing v1.

Reply:
Also, I noticed we need to re-export `Hono` to avoid adding in peer dependencies. So, `/internals` are weird for it.

Reply:
I misunderstood your question probably.
- `/internals` are for someone who might want to experiment something.
- `/hono` is for deploy plugins (actual use case), I wish this could be avoided.

Reply:
Ah, this makes more sense! Thanks for the explanation